lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
```

Comment:
`simple_form.en.yml` is useful to give a dedicated place for adding simple_form settings and we use it in a few projects. If we remove this line we may soon have to at least add another generator to generate that file.